corpus/templates/accounts/edit_profile.html

```diff
@@ -0,0 +1,292 @@
+                                <input type="checkbox" name="hide_linkedin" 
+                                    class="mr-2"
+                                    {% if exec_member.hide_linkedin %}checked{% endif %}>
+                                <span>Hide LinkedIn?</span>
+                            </label>
+                        </div>
+
+                        <!-- GitHub Field -->
+                        <div class="form-control w-full">
+                            <label class="label">
+                                <span class="label-text">GitHub</span>
+                            </label>
+                            {{ executive_member_form.github }}
+                            <label class="flex items-center mt-2 cursor-pointer">
+                                <input type="checkbox" name="hide_github" 
+                                    class="mr-2"
+                                    {% if exec_member.hide_github %}checked{% endif %}>
+                                <span>Hide GitHub?</span>
+                            </label>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            {% endif %}
+
+            <!-- Save Changes Modal -->
+            <div id="saveChangesModal" class="modal" aria-hidden="true">
+                <div class="modal-box">
+                    <h3 class="text-lg font-bold">Confirm Save Changes</h3>
+                    <p>Are you sure you want to save these changes?</p>
+                    <div class="modal-action mt-4 flex justify-center gap-4">
+                        <button type="submit" class="btn btn-primary" id="confirmSaveChanges">Yes, Save</button>
+                        <button type="button" class="btn btn-secondary modal-close">Cancel</button>
+                    </div>
+                </div>
+            </div>
+
+            <!-- Go Back Modal -->
+            <div id="goBackModal" class="modal" aria-hidden="true">
+                <div class="modal-box">
+                    <h3 class="text-lg font-bold">Confirm Go Back</h3>
+                    <p>Are you sure you want to go back? All unsaved changes will be lost!</p>
+                    <div class="modal-action mt-4 flex justify-center gap-4">
+                        <a href="{% url 'accounts_profile' roll_no=exec_member.roll_number %}" class="btn btn-primary">Yes, Go Back</a>
+                        <button type="button" class="btn btn-secondary modal-close">Cancel</button>
+                    </div>
+                </div>
+            </div>
+
+
+            <!-- Buttons Section -->
+            <div class="mt-6 flex justify-center gap-4">
+                <button type="button" class="btn btn-primary" id="openSaveChangesModal">Save Changes</button>
+                <button type="button" class="btn btn-secondary" id="openGoBackModal">Go Back</button>
+            </div>
+
+    </form>
+</div>
+
+
+    <script>
```

Comment:
We have jQuery right? Use that over traditional JS, much easier for us to future-proof and maintain